### DISCUSSION TITLE
Support for utm_term and utm_content

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#515] [Anything Else]
"Would you consider to add Campaign term column in the future? I personal would find it to be pretty useful to track keyword performance."

(request via email)

**Update:** Both utm_term and utm_content are supported and can be found in your Top Sources report

**Comment by rickychilcott** [comment]
This would be awesome. I'm just now getting into using these extra utm parameters for many campaigns and these will be critical for our tracking and analysis.

**Comment by metmarkosaric** [comment]
A request to support utm_keyword sent via email today.

**Comment by ricky11** [comment]
any movement on this?

**Reply by metmarkosaric** [reply]
hi @ricky11! the work on this hasn't started yet and there are no short term plans to introduce it. we are a small team with limited development resources so need to focus on most frequently requested features first.

**Reply by ricky11** [reply]
no worries, i understand, i looked at the blog article about utm and couldn't find how to track utm_terms, any suggestion for paid ad keyword tracking?

**Reply by metmarkosaric** [reply]
depends on what kind of campaign you're running and what's important for you to track but you could use utm_campaign for keywords themselves. something like ?utm_campaign=campaign-keyword or ?utm_campaign=keyword. a bit of a hack but it works

**Comment by benjaminhouy** [comment]
I would love to have this feature as well :).

It would be particularly useful for launches so I could see the total sales for a launch with utm_campaign (e.g, August launch) and then for each eamail with utm_content (e.g, faq-email)

**Comment by metmarkosaric** [comment]
there's a pull request for this now https://github.com/plausible/analytics/pull/1393

if you do have any feedback on how you'd like us to display these without making the dashboard too busy, do share please

**Comment by BlenderDefender** [comment]
The pull request has been merged today. This feature will be added to Plausible Cloud in the next weeks, they're testing it right now. :)

**Comment by metmarkosaric** [comment]
This is now live! Both utm_term and utm_content are supported and can be found in your Top Sources report. Thanks to @BlenderDefender for the contribution!

**Reply by BlenderDefender** [reply]
Could you please mark this reply as answer? That would be great to show that this feature request has been resolved.

**Reply by metmarkosaric** [reply]
not possible to mark replies as answers in the normal thread, only in q&a threads. i've updated the original post too so should be fine

**Comment by benjaminhouy** [comment]
That's awesome! Thanks @BlenderDefender and @metmarkosaric :)

**Reply by metmarkosaric** [reply]
glad you like it!

**Comment by FarisZR** [comment]
Hmm, I can't find the utm_term and utm_content option in the self-hosted version

![image](https://user-images.githubusercontent.com/35614734/193425693-d3fdef16-0d62-4ce1-a957-54812b1bbd3c.png)
